pkgs/top-level/aliases.nix

```
@@ -1030,6 +1030,7 @@ mapAliases ({
   tftp_hpa = tftp-hpa; # added 2015-04-03
   timescale-prometheus = promscale; # added 2020-09-29
   timetable = throw "timetable has been removed, as the upstream project has been abandoned"; # added 2021-09-05
+  tomboy = throw "tomboy is not actively developed anymore and was removed, request packaging tomboy-ng"; # added 2022-01-27
```

```suggestion
  tomboy = throw "tomboy is not actively developed anymore and was removed"; # added 2022-01-27
```
it is very unlikely that this ever gets updated when tomboy-ng is introduced.